package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@buildertrend/eslint-plugin-enterprise-extras",
   "description": "Extra eslint rules for enterprise environments focusing on React and Typescript",
-  "version": "1.1.2",
+  "version": "2.0.0",
```

Comment:
There is some debate here I had with Dan Drews, but I believe new rules fall under "major" revision increases because they can introduce build failures if you are extending the "all"/"recommended" configs.